docs/sources/flow/reference/stdlib/yaml_decode.md

```diff
@@ -0,0 +1,48 @@
+---
+aliases:
+- ../../configuration-language/standard-library/yaml_decode/
+- /docs/grafana-cloud/agent/flow/reference/stdlib/yaml_decode/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/stdlib/yaml_decode/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/stdlib/yaml_decode/
+- /docs/grafana-cloud/send-data/agent/flow/reference/stdlib/yaml_decode/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/stdlib/yaml_decode/
+description: Learn about yaml_decode
+title: yaml_decode
+---
+
+# yaml_decode
+
+The `yaml_decode` function decodes a string representing YAML into a River
+value. `yaml_decode` fails if the string argument provided cannot be parsed as
+YAML.
+
+A common use case of `yaml_decode` is to decode the output of a
+[`local.file`][] component to a River value.
+
+> Remember to escape double quotes when passing YAML string literals to `yaml_decode`.
+>
+> For example, the YAML value `key: "value"` is properly represented by the
+> string `"key: \"value\""`.
```

Comment:
```suggestion
{{< admonition type="tip" >}}
Remember to escape double quotes when passing YAML string literals to `yaml_decode`.

For example, the YAML value `key: "value"` is properly represented by the string `"key: \"value\""`.
{{< /admonition >}}
```

We could format this as a Tip vs just a blockquote. We would have to eventually migrate the other blockquotes in this stdlib reference section, but this can be done in a separate PR.